script.js

```diff
@@ -0,0 +1,259 @@
+
+window.onload = function () {
+  const s = localStorage.getItem('isEng');
+  console.log(s);
+  if (localStorage.length === 0) {
+    return eng();
+  } if (s === 1) {
+    return rus();
+  }
+  return rus();
+};
+function eng() {
+  const span = document.querySelectorAll('.print');
+  span.innerText = '';
+  const en = document.querySelectorAll('.en');
+  const enCap = document.querySelectorAll('.enCap');
+  for (let i = 0, j = 0, n = 0, l = 0;
+    i < enCap.length, j < en.length, n < EN.length, l < EN_CAPS.length;
+    i += 1, j += 1, n += 1, l += 1) {
+    en[i].innerHTML = EN[i];
+    enCap[i].innerHTML = EN_CAPS[i];
+  }
+  const body = document.querySelector('body');
+  body.addEventListener('keydown', (event) => {
+    event.preventDefault();
+    if (event.altKey && (event.ctrlKey || event.metaKey)) {
+      rus();
+      localStorage.clear();
+      localStorage.setItem('isEng', '1');
+    }
+  });
+}
+function rus() {
+  const span = document.querySelectorAll('.print');
+  span.innerText = '';
+  const en = document.querySelectorAll('.en');
+  const enCap = document.querySelectorAll('.enCap');
+  for (let i = 0, j = 0, n = 0, l = 0;
+    i < enCap.length, j < en.length, n < RUS.length, l < RUS_CAPS.length;
+    i += 1, j += 1, n += 1, l += 1) {
+    en[i].innerHTML = RUS[i];
+    enCap[i].innerHTML = RUS_CAPS[i];
+  }
+  const body = document.querySelector('body');
+  body.addEventListener('keydown', (event) => {
+    event.preventDefault();
+    if (event.altKey && (event.ctrlKey || event.metaKey)) {
+      eng();
+      localStorage.clear();
+    }
+  });
+}
+function caps() {
+  const en = document.querySelectorAll('.en');
+  for (let i = 0; i < en.length; i += 1) {
+    if (!((i > 52) || (i === 42) || (i === 41) || (i === 29)
+     || (i === 28) || (i === 13) || (i === 14))) {
+      en[i].classList.toggle('key-style-upper');
+    }
+  }
+}
+function shift() {
+  const en = document.querySelectorAll('.en');
+  const enCap = document.querySelectorAll('.enCap');
+  for (let i = 0, j = 0; i < en.length, j < enCap.length; i += 1, j += 1) {
+    en[i].classList.toggle('hidden');
+    enCap[i].classList.toggle('hidden');
+  }
+}
+const body = document.querySelector('body');
+body.addEventListener('keydown', (event) => {
+  playAudio();
+  event.preventDefault();
+  document.querySelector(`.key[data-code="${event.code}"]`).classList.add('active');
+  printText();
+  if (event.code === 'CapsLock') {
+    document.querySelector(`.key[data-code="${event.code}"]`).classList.toggle('press');
+    caps();
+  }
+  if (event.code === 'ShiftLeft') {
+    if (event.repeat) return;
+    shift();
+  }
+});
+
+body.addEventListener('keyup', (event) => {
+  document.querySelector(`.key[data-code="${event.code}"]`).classList.remove('active');
+
+  if (event.code === 'ShiftLeft') {
+    shift();
+  }
+});
+
+document.querySelectorAll(' .key ').forEach((element) => {
+  element.onmousedown = function (event) {
```

Comment:
One more point here (and this will be also applied for events on lines 189, 199, 207). You add the listener to each of the keys on your keyboard. And as a result, each of your keys has at least 4 listeners. And you have ~65 keys, *4 listeners for each, and as a result on your keyboard ~260 listeners.
For these purposes, it would be better to use event delegation. The main idea of such behaviour is that you will have only 1 `mousedown` event on the parent of the keys, and you will have 1 callback, where you can detect the specific key ( `event.target` ) which was clicked, and depending on the key you will do some specific logic.